plumbum/machines/session.py

```diff
@@ -202,7 +208,10 @@ class ShellSession:
                             is seen, the shell process is killed
     """
 
-    def __init__(self, proc, encoding="auto", isatty=False, connect_timeout=5):
+    def __init__(
+        self, proc, encoding="auto", isatty=False, connect_timeout=5, host=None
```

Comment:
```suggestion
        self, proc, encoding="auto", isatty=False, connect_timeout=5, *, host=None
```

It's harder to change existing args, but new args should be kw-only.